lib/config.js

```diff
@@ -15,6 +15,13 @@ config = JSON.parse(stripJsonComments(configLines));
 config.postgres = new pg.Client('tcp://' + config.postgres);
 config.postgres.connect();
 config.redis = _.extend(redis.createClient(config.redis.port, config.redis.host), config.redis);
+if (config.redis.password) {
+  config.redis.auth(config.redis.password, function (error) {
+    if (error) {
+      console.error('Redis auth failed:', error);
```

Comment:
That is some strange linter setup, why would it complain now with curly brackets, but not beforehand?